src/shared/tpm2-util.c

```diff
@@ -872,7 +935,7 @@ int tpm2_seal(
         rc = sym_Esys_Create(
                         c.esys_context,
                         primary,
-                        ESYS_TR_PASSWORD,
+                        session,
```

Comment:
maybe add an inline comment clarifying why this parameter is passed, i.e. suffix by something short like `/* pass HMAC sesssion to enable parameter encryption */` or so